cypress/support/commands.js

```diff
@@ -23,3 +23,22 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('loginAutomationUser', () => {
```

Comment:
you are not using page object in this functions. You are repeating the same selectors in the 2 methods